tap_hibob/client.py

```diff
@@ -100,3 +100,6 @@ def post_process(self, row: dict, context: Optional[dict]) -> dict:
         """As needed, append or transform raw data to match expected structure."""
         # TODO: Delete this method if not needed.
         return row
+
+    def backoff_max_tries(self) -> int:
```

Comment:
Document link to SDK? 🤔 Should we explain default values and reasoning?

Reply:
Good call! Adding :D